Unfortunately the services EWS is not happy with this. The resultsdb tests are now failing:

```
Downloading Twisted-20.3.0...
Installing Twisted-20.3.0...
Installed Twisted-20.3.0!
Traceback (most recent call last):
  File "runUnittests.py", line 65, in <module>
    sys.exit(main())
  File "runUnittests.py", line 55, in main
    from webkitpy.autoinstalled import buildbot
  File "/Volumes/Data/worker/Services-EWS/build/Tools/Scripts/webkitpy/autoinstalled/buildbot.py", line 27, in <module>
    from webkitpy.autoinstalled import twisted
  File "/Volumes/Data/worker/Services-EWS/build/Tools/Scripts/webkitpy/autoinstalled/twisted.py", line 48, in <module>
    from twisted.protocols.tls import TLSMemoryBIOFactory
  File "/Volumes/Data/worker/Services-EWS/build/Tools/Scripts/libraries/autoinstalled/python-3-x86_64/twisted/protocols/tls.py", line 41, in <module>
    from OpenSSL.SSL import Error, ZeroReturnError, WantReadError
  File "/Volumes/Data/worker/Services-EWS/build/Tools/Scripts/libraries/autoinstalled/python-3-x86_64/OpenSSL/__init__.py", line 8, in <module>
    from OpenSSL import SSL, crypto
  File "/Volumes/Data/worker/Services-EWS/build/Tools/Scripts/libraries/autoinstalled/python-3-x86_64/OpenSSL/SSL.py", line 19, in <module>
    from OpenSSL.crypto import (
  File "/Volumes/Data/worker/Services-EWS/build/Tools/Scripts/libraries/autoinstalled/python-3-x86_64/OpenSSL/crypto.py", line 3271, in <module>
    name="load_pkcs7_data",
TypeError: deprecated() got an unexpected keyword argument 'name'
```

--- a/Tools/Scripts/libraries/webkitcorepy/webkitcorepy/__init__.py
+++ b/Tools/Scripts/libraries/webkitcorepy/webkitcorepy/__init__.py
@@ -83,10 +83,7 @@ else:
 
 AutoInstall.register(Package('pyparsing', Version(2, 4, 7)))
 
-if sys.version_info >= (3, 12):
-    AutoInstall.register(Package('requests', Version(2, 31, 0)))
-else:
-    AutoInstall.register(Package('requests', Version(2, 24)))
+AutoInstall.register(Package('requests', Version(2, 26, 0)))
 
 if sys.version_info >= (3, 0):
     AutoInstall.register(Package('tomli', Version(2, 0, 1), wheel=True))
@@ -94,13 +91,10 @@ if sys.version_info >= (3, 0):
 else:
     AutoInstall.register(Package('setuptools_scm', Version(5, 0, 2), pypi_name='setuptools-scm'))
 AutoInstall.register(Package('socks', Version(1, 7, 1), pypi_name='PySocks'))
-AutoInstall.register(Package('six', Version(1, 15, 0)))
+AutoInstall.register(Package('six', Version(1, 16, 0)))
 AutoInstall.register(Package('tblib', Version(1, 7, 0)))
 
-if sys.version_info >= (3, 12):
-    AutoInstall.register(Package('urllib3', Version(2, 0, 4)))
-else:
-    AutoInstall.register(Package('urllib3', Version(1, 25, 10)))
+AutoInstall.register(Package('urllib3', Version(1, 26, 17)))
 
 AutoInstall.register(Package('wheel', Version(0, 35, 1)))
 AutoInstall.register(Package('whichcraft', Version(0, 6, 1)))

--- a/Tools/Scripts/webkitpy/__init__.py
+++ b/Tools/Scripts/webkitpy/__init__.py
@@ -35,7 +35,7 @@ from webkitcorepy import AutoInstall, Package, Version
 AutoInstall.set_directory(os.path.join(libraries, 'autoinstalled', 'python-{}-{}'.format(sys.version_info[0], platform.machine())))
 
 if sys.version_info >= (3, 7):
-    AutoInstall.register(Package('pylint', Version(2, 6, 0)))
+    AutoInstall.register(Package('pylint', Version(2, 13, 9)))
     AutoInstall.register(
         Package("pytest", Version(7, 2, 0),
                 implicit_deps=["attr", "pluggy", "iniconfig"]
@@ -73,7 +73,6 @@ AutoInstall.register(Package('configparser', Version(4, 0, 2), implicit_deps=['p
 AutoInstall.register(Package('contextlib2', Version(0, 6, 0)))
 AutoInstall.register(Package('coverage', Version(5, 2, 1)))
 AutoInstall.register(Package('funcsigs', Version(1, 0, 2)))
-AutoInstall.register(Package('genshi', Version(0, 7, 3), pypi_name='Genshi'))
 AutoInstall.register(Package('html5lib', Version(1, 1)))
 AutoInstall.register(Package('iniconfig', Version(1, 1, 1)))
 AutoInstall.register(Package('mechanize', Version(0, 4, 5)))
@@ -84,13 +83,24 @@ AutoInstall.register(Package('mozterm', Version(1, 0, 0)))
 AutoInstall.register(Package('pluggy', Version(0, 13, 1)))
 AutoInstall.register(Package('py', Version(1, 11, 0)))
 AutoInstall.register(Package('pycodestyle', Version(2, 5, 0)))
-AutoInstall.register(Package('pyfakefs', Version(3, 7, 2)))
+
+if sys.version_info >= (3, 7):
+    AutoInstall.register(Package('pyfakefs', Version(5, 2, 4)))
+else:
+    AutoInstall.register(Package('pyfakefs', Version(3, 7, 2)))
+
 AutoInstall.register(Package('scandir', Version(1, 10, 0)))
-AutoInstall.register(Package('soupsieve', Version(1, 9, 6)))
+
+if sys.version_info >= (3, 6):
+    AutoInstall.register(Package('soupsieve', Version(2, 2, 1)))
+else:
+    AutoInstall.register(Package('soupsieve', Version(1, 9, 6)))
+
 if sys.version_info < (3, 8):
     AutoInstall.register(Package('selenium', Version(3, 141, 0)))
 else:
     AutoInstall.register(Package('selenium', Version(4, 12, 0)))
+
 AutoInstall.register(Package('toml', Version(0, 10, 1), implicit_deps=['pyparsing']))
 AutoInstall.register(Package('wcwidth', Version(0, 2, 5)))
 AutoInstall.register(Package('webencodings', Version(0, 5, 1)))

--- a/Tools/Scripts/webkitpy/conftest.py
+++ b/Tools/Scripts/webkitpy/conftest.py
@@ -26,6 +26,8 @@ import sys
 
 import pytest
 
+from webkitcorepy import AutoInstall
+
 
 def pytest_configure(config):
     config.addinivalue_line("markers", "serial: tests that must be run in serial")
@@ -104,3 +106,7 @@ def pytest_collection_modifyitems(config, items):
         for item in items:
             if "slow" in item.keywords:
                 item.add_marker(skip_slow)
+
+
+def pytest_collection_finish(session):
+    AutoInstall.install_everything()

